concordium-node/Cargo.lock

```diff
@@ -535,15 +544,17 @@ dependencies = [
 
 [[package]]
 name = "concordium-contracts-common"
-version = "4.0.0"
+version = "4.1.0"
 dependencies = [
- "base58check",
+ "bs58",
  "chrono",
  "concordium-contracts-common-derive",
  "fnv",
  "hashbrown 0.11.2",
```

Comment:
base58check is still listed as a dependency of concordium-node. Please resolve that as well.